Content.Shared/_DV/CosmicCult/CosmicCult.Events.cs

```diff
@@ -0,0 +1,18 @@
+using Robust.Shared.Serialization;
+
+namespace Content.Shared._DV.CosmicCult;
+
+[Serializable, NetSerializable]
+public sealed partial class CosmicSiphonIndicatorEvent : EntityEventArgs
+{
+    public NetEntity Target = new();
```

Comment:
The constructor always sets this
```suggestion
    public NetEntity Target;
```